viewing_party/party.py

```diff
@@ -1,23 +1,147 @@
 # ------------- WAVE 1 --------------------
-
+from ast import AnnAssign
+from curses import use_default_colors
+from operator import ne
+import re
+import copy
+# from tkinter import N
 def create_movie(title, genre, rating):
-    pass
+    dic = {}
+    dic["title"] = title
+    dic["genre"] = genre
+    dic["rating"] = rating
+    if dic["title"] == None or dic["genre"] == None or dic["rating"] == None:
+        return None
+    return dic
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(janes_data, MOVIE_TITLE_1):
+
+    if len(janes_data["watched"]) == 0:
+        temp_list = janes_data["watchlist"].pop()
+        janes_data["watched"].append(temp_list)
+        return janes_data
+    elif len(janes_data["watchlist"]) > 1:
+        temp_list = janes_data["watchlist"].pop()
+        janes_data["watched"].append(temp_list)
+        return janes_data 
+    else:
+        return janes_data
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
 
+def get_watched_avg_rating(janes_data):
+    global average
+    if janes_data["watched"] == []:
+        average = len(janes_data["watched"])

```

Comment:
To speed up efficiency here, you could also `return 0` since you know the return value you want and don't need to execute the further lines of code in this function.